modules/monitoring-choosing-a-metrics-collection-profile.adoc

```diff
@@ -64,4 +58,4 @@ data:
       collectionProfile: *minimal*
 ----
 
-. Save the file to apply the changes. The pods affected by the new configuration restart automatically.
\ No newline at end of file
+. Save the file to apply the changes. The new configuration is applied automatically.
```

Comment:
@juzhao  When I tested this, this change did not trigger pods to redeploy so I am not including the sentence about the redeployment here.

Reply:
right, no pods restarted